package.json

```diff
@@ -30,7 +30,7 @@
     "integration:headful": "concurrently -k -s first \"npm:blockchain\" \"npm:integration:local\" \"npm:integration:metamask\"",
     "lint": "eslint ./src --ext .js,.ts,.tsx --max-warnings 0",
     "lint:fix": "npm run lint -- --fix",
-    "prep": "run-s lint:fix test integration:headless integration:headful",
+    "prep": "run-s lint:fix test integration:headless:dev integration:headful",
```

Comment:
Just curious, what's the difference in the test when running `integration:headless` and `integration:headless:dev`?

Reply:
not too sure but the config different for `:dev` and `:prod`. u think should change to prod instead?

Reply:
https://github.com/TradeTrust/tradetrust-website/pull/628

Reply:
Ahh... Okay, then having `dev` here as what @isaackps has it seems to make sense to me.👍

Reply:
`prep` was a naive task i added in long time back to run through all “ci” tasks before committing.

never really used it anymore. is can remove one.

Reply:
actually i still use it leh, cos i feel that it is very useful. I usually run prep 1st before pushing it up.